### PR TITLE
Update AUTHORS file

### PR DESCRIPTION
**Pull request by tasleson**
Add everyone that has committed at least 1 commit to this file
to encourage others to contribute too.

Note: Info collected with `git shortlog -se`

Signed-off-by: Tony Asleson <tasleson@redhat.com>

### CODE REVIEW

**Comment by vpodzime** [comment]
I'm not really sure this change makes much sense. Not that I don't want to make someone's contributions look like less important, but what's the purpose of having a generated file in the repo? Everybody can run the command to get an up to date list, we are not in the age of having tarballs on *ftp* servers after all. So I think we should either remove the file or just have a few up-to-date people listed there.

**Comment by tasleson** [comment]
Not all aspects of the file can be auto-generated.  The file should likely include a section with a current list of people with commit access too.

My philosophy is if anyone takes the time to submit a change that is accepted into a project, that they should be known and credited.  We are really trying to give credit to everyone, to encourage others to contribute too.  Otherwise, what message are we conveying?  *"Oh, your name can be in this file only after you author at least some N number of commits."*  I'm less worried about giving credit to frequent contributors than I am about giving credit to someone that just contributed.  Without new people contributing, a project just atrophies overtime.  IMHO we want to encourage new people to help, giving them credit sooner than later is a way to facilitate that.

**Comment by vpodzime** [comment]
@tasleson, that sounds fair to me. Please suggest a mechanism for keeping the file up to date, though. Shall we regenerate it when doing releases maybe?

**Comment by tasleson** [comment]
@vpodzime Release time sounds good to me.

**Comment by vpodzime** [comment]
> @vpodzime Release time sounds good to me.

Then please update the ``HACKING`` file.

File: HACKING [hunk 1]
```diff
@@ -32,6 +32,8 @@ Checklist for making a release:
 
  - Update the NEWS file
 
+ - Update the AUTHORS file if we have any new authors since last release.
```
Comment (vojtechtrefny) [inline HACKING]: Please add the git command to create the list here. I don't want to google it for every release :-)

**Comment by tasleson** [comment]
Added blurb about git command as requested by @vojtechtrefny

**Comment by tasleson** [comment]
@vpodzime Squashed as requested

**Pull request merged**